src/test/java/org/kainos/ea/db/JobRoleDaoTest.java

```diff
@@ -0,0 +1,78 @@
+package org.kainos.ea.db;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.api.JobRoleService;
+import org.kainos.ea.cli.JobRole;
+import org.kainos.ea.client.FailedToGetJobRoles;
+import org.kainos.ea.client.JobRolesNotFoundException;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.xml.crypto.Data;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyString;
+
+@ExtendWith(MockitoExtension.class)
+public class JobRoleDaoTest {
+    private DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+    private Connection connection = Mockito.mock(Connection.class);
+    private PreparedStatement statement = Mockito.mock(PreparedStatement.class);
+    private ResultSet resultSet = Mockito.mock(ResultSet.class);
+    private JobRoleDao jobRoleDao = new JobRoleDao(databaseConnector);
+    @Test
+    void getJobRoles_ShouldReturnRoles_WhenDatabaseReturnsRoles() throws SQLException {
+        String query = "SELECT RoleID, Name, Job_Spec, Responsibilities, Sharepoint_Link, " +
```

Comment:
Could you make the query static in the class and then reference it in the method and the tests? It would make it less likely to break with changes

Reply:
Turns out don't even need the query so just deleted it